src/Cassandra/Mapping/TypeConversion/TypeConverter.cs

```diff
@@ -199,8 +199,7 @@ TResult ChangeType(TSource a)
 
             return (Func<TSource, TResult>) converter;
         }
-
-
+        
```

Comment:
As long as we don't leak it in the root namespace, we are safe, but maybe we can create a ticket to extract a "base class" and move it to the root namespace, while continue to expose it in the Mapper.

Reply:
What is this comment referring to? I'm confused 😄

Reply:
oh sorry, I'm referring to `TypeConverter` class.

`TypeConverter` is being used by UDTs and Graph but it's on the `Mapping` namespace.

Reply:
Ah I understand now, yeah that makes sense I'll create a ticket.